src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -15,7 +15,9 @@ import FiltersSidebarTemplate from './filters-sidebar.hbs';
 
 import './filters-sidebar.scss';
 
-const groupLabelView = intl.patients.sidebar.filters.filtersSidebarViews.groupLabelView;
+const i18n = intl.patients.sidebar.filters.filtersSidebarViews;
```

Comment:
Just added this to make things more concise when accessing `intl.patients.sidebar.filters.filtersSidebarViews` values from the `/i18n/en-US.yml` file.